python-ecosys/cbor2/manifest.py

```diff
@@ -1,3 +1,3 @@
-metadata(version="0.1.0", pypi="cbor2")
+metadata(version="0.1.1", pypi="cbor2")
```

Comment:
This will need more than a patch release bump.

I suggest to bump the major release to 1.0.0 and not even include the backwards compatible `decoder.py` and `encoder.py` files.

Otherwise, if you think backwards compatibility (with a message printed...) is needed, then I suggest version 0.2.0.

Reply:
No it's not needed for our use case.